envVariables.js

```diff
@@ -61,127 +61,46 @@ let enviromentVariables = {
     "optional" : false,
     "default" : "ON"
   },
-  "CLOUD_STORAGE" : {
-    "message" : "Enable/Disable cloud services",
+  // cloud service variables
+  "CLOUD_STORAGE_PROVIDER" : {
+    "message" : "Require cloud storage provider",
     "optional" : false
   },
-  "GCP_PATH" : {
-    "message" : "Required Gcp path",
-    "optional" : true,
-    "requiredIf" : {
-      "key": "CLOUD_STORAGE",
-      "operator": "EQUALS",
-      "value" : "GC"
-    }
+  "CLOUD_STORAGE_BUCKETNAME" : {
+    "message" : "Require client storage bucket name",
+    "optional" : false
   },
-  "DEFAULT_GCP_BUCKET_NAME" : {
-    "message" : "Required Gcp bucket name",
-    "optional" : true,
-    "requiredIf" : {
-      "key": "CLOUD_STORAGE",
-      "operator": "EQUALS",
-      "value" : "GC"
-    }
+  "CLOUD_STORAGE_SECRET" : {
+    "message" : "Require client storage provider identity",
+    "optional" : false
   },
-  "AZURE_ACCOUNT_NAME" : {
-    "message" : "Required Azure Account name",
-    "optional" : true,
-    "requiredIf" : {
-      "key": "CLOUD_STORAGE",
-      "operator": "EQUALS",
-      "value" : "AZURE"
-    }
+  "CLOUD_STORAGE_ACCOUNTNAME" : {
+    "message" : "Require client storage account name",
+    "optional" : false
   },
-  "AZURE_ACCOUNT_KEY" : {
-    "message" : "Required Azure Account key",
-    "optional" : true,
-    "requiredIf" : {
-      "key": "CLOUD_STORAGE",
-      "operator": "EQUALS",
-      "value" : "AZURE"
-    }
+
```

Comment:
@Prajwal17Tunerlabs remove unwanted new lines please

Reply:
removed unwanted lines from the code